versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java

```diff
@@ -62,16 +63,17 @@ protected AbstractNamespaceValidation(VersionStore store) {
   }
 
   static Stream<ContentKey> contentKeys() {
-    return Stream.of(
-        ContentKey.of("ns", "table"),
-        ContentKey.of("ns", "table"),
-        ContentKey.of("ns2", "ns", "table"),
-        ContentKey.of("ns2", "ns", "table"));
+    return Stream.of(ContentKey.of("ns", "table"), ContentKey.of("ns2", "ns", "table"));
   }
 
   @ParameterizedTest
   @MethodSource("contentKeys")
   void commitWithNonExistingNamespace(ContentKey key) throws Exception {
+
+    assumeTrue(
+        key.getElementCount() == 2 || !isNewStorageModel(),
```

Comment:
this leaves only one parameter for this method, why bother with `@ParameterizedTest`?

Reply:
We are still testing the multiple missing namespaces case for the old model here.

Reply:
`!isNewStorageModel()` makes sense, but do we have more one parameter?

Reply:
sorry, misinterpreted :facepalm: